<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SYSID</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin"><number>16</number></property>
    <property name="topMargin"><number>16</number></property>
    <property name="rightMargin"><number>16</number></property>
    <property name="bottomMargin"><number>16</number></property>

    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontal_layout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QWidget" name="buttons_container">
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>



          <layout class="QVBoxLayout" name="interfaces_buttons_layout">


           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_info">
             <property name="text">
              <string>Choose one of the following applications:</string>
           </property>
            </widget>
           </item>




           <item>
            <widget class="QPushButton" name="sysid_interface_orbit_button">
             <property name="text"><string>SysID - Orbit</string></property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch><verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>

           <item>
            <widget class="QPushButton" name="sysid_interface_dispersion_button">
             <property name="text"><string>SysID - Dispersion</string></property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch><verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>

           <item>
            <widget class="QPushButton" name="sysid_interface_wakefield_button">
             <property name="text"><string>SysID - Wakefield</string></property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch><verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>



           <item>
            <widget class="QPushButton" name="compute_matrix_button">
             <property name="text"><string>Compute Response Matrix</string></property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch><verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bba_interface_button">
             <property name="text"><string>Beam Based Alignment</string></property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch><verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="emittance_interface_button">
             <property name="text"><string>Emittance Measurement</string></property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch><verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="knobs_interface_button">
             <property name="text"><string>Knobs</string></property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch><verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>



          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
